doc/HOWTO_Import_Geodata.md

```diff
@@ -0,0 +1,8 @@
+# How to Import the Third Party Geodata
+
+The Geodata zip contains 2 sql files, a small one with community names, and a large one which includes streets.
+
+1. Copy the smaller sql file into /build/database.
+2. Rename it to `00.[FILENAME].sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```

Comment:
```suggestion
2. Rename it to `00.Geodata.sql`. This is needed because Docker runs the files in order, and the Geodata import needs to run before the Schema.
```
I'm all for indicating that a file name is irrelevant, but using a placeholder here might trip up inexperienced developers who might name that file literally `00.[FILENAME].sql`